at_client/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.0.37
+- fix: Refrain sending client config changes to server as server changes are deployed
```

Comment:
fix : revert sending client config changes to server

Reply:
Updated the changelog